### PR TITLE
Improve the registration page:

### PR DESCRIPTION
**Pull request by mchv**
- Display developer and commercial differences side by side
- Display forms only when user has selected a tiers
- Link to the appropriate page on mashery if the user has a mashery account

screen shot 2014-07-02 at 17 46 58
screen shot 2014-07-02 at 17 10 08
screen shot 2014-07-02 at 17 10 17
screen shot 2014-07-02 at 17 10 26

### CODE REVIEW

File: access/index.html [hunk 1]
```diff
@@ -25,44 +25,58 @@
...
-			</form>
-		</div>
+        <div id="commercialBox" class="box">
+            <h1>Commercial</h1>
+            <ul>
+                <li>Enquire</li>
+                <li>Up to unlimited calls per second</li>
+                <li>Up to unlimited calls per day</li>
+                <li>Access to article text, images, audio and videos</li>
+                <li>Access to entire Guardian content</li>
```
Comment (rrees) [inline access/index.html]: Think "archive" or "content store" might sound a bit better

File: access/index.html [hunk 2]
```diff
@@ -25,44 +25,58 @@
...
-	          	<button type="submit" onclick="window.location='http://guardian.mashery.com'">GO TO MASHERY</button>
-	          	
-	          </div>
-			</form>
-		</div>
+        <div id="commercialBox" class="box">
+            <h1>Commercial</h1>
+            <ul>
+                <li>Enquire</li>
+                <li>Up to unlimited calls per second</li>
```
Comment (rrees) [inline access/index.html]: Can you really do unlimited calls per second? Thousands might be okay here

Comment (jennysivapalan) [inline access/index.html]: Agreed, we could change this and the below bullet point to highlight that custom levels can be set.

**Comment by jennysivapalan** [comment]
I like the changes but I'm not a fan of the toggle without some labelling on. I'd either just have a register and sign in buttons. Alternatively if you do like the toggle then have a label to it?

**Comment by mchv** [comment]
@rrees @jennysivapalan  thanks for feedback, I have updated wording. I will add a label for the toggle.

**Comment by nicl** [comment]
Can we get a designer to have a look and maybe revamp it. I know it is late in the day, but stepping back, my honest perspective is that the visuals are just not there yet.

Some specifics:
- the toggle is confusing and unnecessary
- the background gradient and colour is too strong
- the design makes no effort to be consistent with NG (typography, colours, spacing, etc.)

In general, I'd especially like it to be more consistent with the NG design-scheme. Alternatively, we could perhaps revisit design that Oliver created for the developer site as part of the last hackday.

**Comment by mchv** [comment]
> Can we get a designer to have a look and maybe revamp it. I know it is late in the day, but stepping back, my honest perspective is that the visuals are just not there yet.

Thanks for your feedback, I personally find it nice, but it's entirely subjective.
However I could explain several choices, and based on your points we could improve it without to revamp it. 

> the toggle is confusing and unnecessary

The toggle is used to change link and button text. Another option is to provide two links.
Regarding checking confusion, I have updated it to include a label

screen shot 2014-07-02 at 18 22 10

> the background gradient and colour is too strong

I used the to blues used on NG to create the gradient. We could change the stop colour to make it more weak.

> the design makes no effort to be consistent with NG (typography, colours, spacing, etc.)

I agree on typography but not on the others. I have tried to reuse colours.
Regarding typography, I find `EgyptianText`font difficult to read on screen.

> In general, I'd especially like it to be more consistent with the NG design-scheme

Strong consistency with NG was not my main aim, but we could improve on this topic.

**Comment by mchv** [comment]
I have updated the branch with some modifications thanks to @nicl feedback.

screen shot 2014-07-03 at 14 09 50
screen shot 2014-07-03 at 14 25 41
screen shot 2014-07-03 at 14 25 57

**Comment by gidsg** [comment]
Other than my [comment](https://github.com/guardian/open-platform-site/pull/7#commitcomment-6886397) :+1:

**Comment by mchv** [comment]
Thanks @gidsg. I will merge as it and let you update directly on `gh-pages` branch.

**Pull request merged**